packages/sparse-ints/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/sparse-ints",
-  "version": "0.1.22",
+  "version": "0.1.23",
   "description": "Share code across vat boundaries",
```

Comment:
sparse-ints description looks like copy-pasta

Reply:
Hmm.  I'll probably drive-by fix it next time I think of it.